### ISSUE TITLE
Connecting Redis to Munin (cPanel server)

### ISSUE THREAD

**Issue by Yossifsolman** [munin-monitoring/munin#1490]
Hey, have an option to connect Redis to Munin?
![image](https://user-images.githubusercontent.com/104964400/189473655-07a6b3fa-76d7-4315-9e98-05b185757f4a.png)

**Comment by smokris** [comment]
There are a few Redis plugins in the gallery: https://gallery.munin-monitoring.org/keywords/redis/

**Comment by Yossifsolman** [comment]
Yeah, i see that before, no any doc's or instruction… little scary to install it on cPanel server without any guidance.
